core/src/test/kotlin/generator/TestRunner.kt

```diff
@@ -0,0 +1,101 @@
+package generator
+
+import org.jetbrains.dokka.*
+import org.jetbrains.dokka.utilities.DokkaConsoleLogger
+import org.jetbrains.dokka.utilities.DokkaLogger
+import java.io.File
+import java.nio.charset.Charset
+import java.nio.file.Files
+import java.nio.file.Path
+import java.nio.file.Paths
+
+
+object TestRunner {
+    private var testDir: Path? = null
```

Comment:
Looks like we don't need this at all, since it's only used in one method locally

Reply:
I've changed it to lazy init - main tests directory is common for all tests to make it easier to clean up after them.